cle/backends/java/apk.py

```diff
@@ -0,0 +1,50 @@
+from .. import register_backend
+
+from .soot import Soot
```

Comment:
No blank line between the two imports above - they are all relative imports from the current module.